coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
The usecases that I encountered worked with classes only, so I think it was the original purpose. I think you cannot have two methods with the same name, one as a class method and the other as regular one.